src/plugins/intel_cpu/tests/functional/CMakeLists.txt

```diff
@@ -19,18 +19,33 @@ else()
     set(EXCLUDED_SOURCE_PATHS ${CMAKE_CURRENT_SOURCE_DIR}/extension ${CMAKE_CURRENT_SOURCE_DIR}/shared_tests_instances/onnx)
 endif()
 
-if(NOT X86_64)
+if(X86_64)
+    list(APPEND EXCLUDED_SOURCE_PATHS
+    ${CMAKE_CURRENT_SOURCE_DIR}/single_layer_tests/instances/arm
+    ${CMAKE_CURRENT_SOURCE_DIR}/subgraph_tests/arm)
+else()
```

Comment:
@EgorDuplensky I beleive we also sohuld split the branche (one logic for ARM and another for WASM, RISC etc).